manual/README.md

```diff
@@ -0,0 +1,62 @@
+
+The resources in REMS do not necessarily have to be in electronic data sets. It is possible to manage access to basically anything as long as it is identified by an identifier. This includes for example research datasets and biological samples.
+
+REMS is a service developed by [CSC – IT Center for Science Ltd.](https://www.csc.fi/en/home) and [Nitor](https://www.nitor.com/en).
+
+You can test how REMS works [using our demo environment](https://remsdemo.csc.fi/).
+
+## Organisations who are already using REMS
+
+[THL, Finnish institute for health and welfare](https://thl.fi/en/web/thl-biobank/for-researchers/application-process/thl-biobank-application-portal), uses REMS to manage access to biological samples and sample related data.
+
+According to THL, the benefits of REMS include:
+
+- Adaptive, versatile tools for controlling resources
+- Administrators have full control of the application in all stages of the process
+- Different language versions
+- Possibility to add attachments during the application process
+- All the research group members can see the application
+
+In addition to THL, [the Language Bank of Finland](https://www.kielipankki.fi/language-bank/) and [Findata](https://www.findata.fi/en/services/data-requests/) are also using REMS.
+
+REMS is developed in close cooperation with our customers. You can follow the agile development of REMS from our [public Github project board](https://github.com/CSCfi/rems/projects/1).
+
+## Why REMS
+
+It is not always possible to share data openly. Sometimes data can contain sensitive personal data that restricts its use. However, sensitive data is also very important for research, which is why accessing it needs to be easy but secure. With REMS, **managing access to sensitive data is simple but secure**.
+
+![REMS features](img/features.png)
+
+## Who can use REMS
+
+REMS is useful for individual researchers who want to get access to certain resources, as well service providers who want to manage access to their material.
+
+Benefits for the resource owner:
+
+- **A dynamic way to handle applications and manage access rights**: The data owner can define for example, how involved they want to be in the approval process and how the process proceeds.
+- **An audit trail of all committed actions and enhanced reporting tools to improve application traceability**: you can look at the application’s full processing history
+- **Customisable application process**: ability to create and modify the application forms
+- **Automation**: Some functions, such as approving and rejecting applications, can be automated to reduce unnecessary workload.
+
+If you want to start using REMS to manage access rights to your resources, contact us at rems-support@csc.fi.
```

Comment:
This could possibly be a link but I suppose it's better not for less spam?

Reply:
Knowingly chose not to make it a link.